code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -85,6 +114,39 @@
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
 	explosion(get_turf(src.loc),-1,0,2)
+	if(shrapnel_list.len > 0)
+		var/i
+		var/atom/target
+		var/list/bodyparts = list("head","chest","groin","l_arm","r_arm","l_hand","r_hand","l_leg","r_leg","l_foot","r_foot")
```

Comment:
There's got to be a better way to do this, I think you could just use something like `ran_zone(LIMB_CHEST,75)` which is what throwing items does